cura/CuraApplication.py

```diff
@@ -2011,6 +2015,33 @@ def shouldShowWhatsNewDialog(self) -> bool:
         show_whatsnew_only = has_active_machine and has_app_just_upgraded
         return show_whatsnew_only
 
+    @pyqtSlot(result = bool)
+    def shouldShowMarketPlacePluginBanner(self) -> bool:
+        return self._preferences.getValue("cura/market_place_show_plugin_banner")
+
+    @pyqtSlot(result = bool)
```

Comment:
I think it's better to just use the direct bindings that we have for preferences in qml. You can gte the value like this:
UM.Preferences.getValue("cura/market_place_show_material_banner")

The same also goes for the hide / setting

Reply:
https://github.com/Ultimaker/Cura/pull/10948/commits/e5c8d5456de7e288905ccb92d7673ac6f13590ff